code/controllers/subsystem/processing/quirks.dm

```diff
@@ -32,6 +34,13 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 		list("Social Anxiety", "Mute"),
 	)
 
+	/// A list of quirks that cannot be used by specific species. Format: species = list(quirk1, quirk2)
+	var/static/list/species_quirk_blacklist = list(
+		/datum/species/plasmaman = list(
+			/datum/quirk/blooddeficiency,
+		),
+	)
```

Comment:
```suggestion
	var/static/list/species_quirk_blacklist = list(
		/datum/species/plasmaman = list("Blood Deficiency"),
	)
```

Shouldn't we be using strings to be consistent with the quirk blacklist?

Reply:
I had it this way in case any name changes to the quirk won't break it.

Reply:
https://github.com/tgstation/tgstation/blob/050fcfcded861b379f64237e1f7bef3a67f0d7e6/code/controllers/subsystem/processing/quirks.dm#L19-L33

Any name change is probably going to break it in a lot of places.  I remember when I wanted to rename nearsighted quirk to bad vision and mothblocks told me it would affect save preferences.